CHANGELOG.rst

```diff
@@ -2,6 +2,24 @@
 Changelog
 =========
 
+2.0.6 (2023-04-??)
+==================
+
+
+.. warning:: Manual intervention required if analytics tools are enabled
+
+   When enabling analytics tools, CSP directives were automatically added to the admin under  **Configuratie** > **CSP settings**.
+   The directive ``connect-src <domain of the analytic tool>`` was causing forms to no longer load.
+
+   In order to fix this issue:
+
+   1. Go to  **Configuratie** > **CSP settings**
+   2. Delete the directive ``connect-src <domain of the analytic tool>``
```

Comment:
Weren't/aren't there other directives that should be removed? I think it should be "remove anything different than `default-src`"

PR: https://github.com/open-formulieren/open-forms/pull/2843/files